Documentation/configuration/argocd-issues.rst

```diff
@@ -0,0 +1,35 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _argocd_issues:
+
+***********************************
+Issues Deploying Cilium with ArgoCD
+***********************************
+
+There have been a lot of reports from users hitting issues with ArgoCD. This documentation 
+page outlines some of the known issues.
+
+Known issues with solutions
+===========================
+The following are some known issues with ArgoCD that have solutions. Click on them to 
+see their solutions on GitHub.
+
+- `ArgoCD deletes CustomResourceDefinitions that Cilium uses to implement networking and security <https://github.com/cilium/cilium/issues/17349>`__.
+- `ciliumid not found  <https://github.com/cilium/cilium/issues/17614>`__.
+- `Out-of-sync issues for cilium using ArgoCD <https://github.com/cilium/cilium/issues/14550>`__.
+- `CiliumEndpoint missing for a pod <https://github.com/cilium/cilium/issues/17047>`__.
+- `hubble-relay: failed to create gRPC client <https://github.com/cilium/cilium/issues/16361>`__.
+- `Add a configmap checksum to automatically roll daemonset <https://github.com/cilium/cilium/issues/14331>`__.
+- `Cilium does not recognise identity thus got policy denied <https://github.com/cilium/cilium/issues/14284>`__.
+- `Intermittent 504 error from service when using ingress network policy <https://github.com/cilium/cilium/issues/13240>`__.
```

Comment:
I'd use this (but it's your call)

```suggestion
- ArgoCD deletes CustomResourceDefinitions that Cilium uses to implement networking and security (:gh-issue:`17349`)
- ciliumid not found  (:gh-issue:`17614`)
- Out-of-sync issues for cilium using ArgoCD (:gh-issue:`14550`)
- CiliumEndpoint missing for a pod (:gh-issue:`17047`)
- hubble-relay: failed to create gRPC client (:gh-issue:`16361`)
- Add a configmap checksum to automatically roll daemonset (:gh-issue:`14331`)
- Cilium does not recognise identity thus got policy denied (:gh-issue:`14284`)
- Intermittent 504 error from service when using ingress network policy (:gh-issue:`13240`)
```

Also maybe sort the issues either by number or alphabetically?